src/types.ts

```diff
@@ -36,11 +36,20 @@ export interface Column<TRow, TSummaryRow = unknown> {
   sortable?: boolean;
   /** Sets the column sort order to be descending instead of ascending the first time the column is sorted */
   sortDescendingFirst?: boolean;
-  // TODO: Should we return an object to make it more powerful?
-  // { isEditable?: boolean, initialInput?: unknown } | undefined
-  unsafe_onCellInput?: (event: React.KeyboardEvent<HTMLDivElement>, row: TRow) => unknown;
   /** Editor to be rendered when cell of column is being edited. If set, then the column is automatically set to be editable */
   editor?: React.ComponentType<EditorProps<TRow[keyof TRow], TRow, TSummaryRow>>;
+  editor2?: React.ComponentType<Editor2Props<TRow, TSummaryRow>>;
+  editorOptions?: {
+    /** Default: true for editor1 and false for editor2 */
+    createPortal?: boolean;
+    /** Default: false */
+    editOnClick?: boolean;
+    // TODO: Do we need these options
+    // editOnDoubleClick?: boolean;
+    // commitOnScroll?: boolean;
+    // commitOnOutsideClick?: boolean;
+    onCellInput?: (event: React.KeyboardEvent<HTMLDivElement>) => void; // Prevent default to cancel editing
```

Comment:
Should we rename it to `onKeyDown` or `onCellKeyDown`?

Reply:
It's only used when the editor is closed right? Then `onCellKeyDown` makes more sense.